tasks/make_zipapp.py

```diff
@@ -17,16 +17,17 @@ def main():
     else:
         dest = os.path.join(args.root, "virtualenv.pyz")
 
-    bio = io.BytesIO()
-    with zipfile.ZipFile(bio, "w") as zipf:
-        filenames = ["LICENSE.txt", "virtualenv.py"]
-        for whl in os.listdir(os.path.join(args.root, "virtualenv_support")):
-            filenames.append(os.path.join("virtualenv_support", whl))
+    filenames = {"LICENSE.txt": "LICENSE.txt", os.path.join("src", "virtualenv.py"): "virtualenv.py"}
```

Comment:
I suspect this zipapp is nonfunctional -- the tests are probably only passing because `virtualenv`'s dependencies are installed in those environments

Reply:
this is one of those tbd https://github.com/pypa/virtualenv/issues/1464 things